### PR TITLE
Fix step 3

### PR DESCRIPTION
**Pull request by alejandronanez**

### CODE REVIEW

File: src/example/solution.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ function init(initialState) {
 
 function tick({ hours, minutes, seconds }) {
   if (hours > 0 && (minutes <= 0 && seconds <= 0)) {
-    return { hours: hours - 1, minutes: 59, seconds: 59};
+    return { hours: hours - 1, minutes: 59, seconds: 59 };
```
Comment (alejandronanez) [inline src/example/solution.js:14]: Prettier doing its thing

File: src/example/solution.js [hunk 2]
```diff
@@ -77,19 +77,24 @@ function Timer() {
...
-    clearInterval(timer);
+    clearInterval(timer.current);
   }
 
   function handleReset() {
     dispatch({ type: 'reset', payload: initialState });
   }
 
-  if (state.isRunning && state.hours <= 0 && state.minutes <= 0 && state.seconds <= 0) {
+  if (
```
Comment (alejandronanez) [inline src/example/solution.js:92]: Prettier doing its thing again

File: src/example/solution.js [hunk 3]
```diff
@@ -124,22 +129,14 @@ function Timer() {
       >
         Start
       </button>
-      <button
-        onClick={handleStop}
-        className="stop"
-        disabled={!isRunning}
-      >
+      <button onClick={handleStop} className="stop" disabled={!isRunning}>
```
Comment (alejandronanez) [inline src/example/solution.js:132]: Prettier doing its thing again

File: src/example/solution.js [hunk 4]
```diff
@@ -124,22 +129,14 @@ function Timer() {
...
         Stop
       </button>
-      <button
-        onClick={handleReset}
-        className="reset"
-        disabled={isRunning}
-      >
+      <button onClick={handleReset} className="reset" disabled={isRunning}>
```
Comment (alejandronanez) [inline src/example/solution.js:135]: More prettier stuff

File: src/example/solution.js [hunk 5]
```diff
@@ -65,7 +65,7 @@ const initialState = {
 
 function Timer() {
   const [state, dispatch] = useReducer(reducer, initialState, init);
-  let timer;
+  const timer = useRef(null);
```
Comment (alejandronanez) [inline src/example/solution.js:68]: If you want to _keep alive_ a value inside a hooks after each cycle, you have to use `useRef`,

Per [React docs](https://reactjs.org/docs/hooks-reference.html#useref):

> Essentially, useRef is like a “box” that can hold a mutable value in its .current property.

Comment (simonhoyos) [reply src/example/solution.js:68]: Great, thanks @alejandronanez

**Pull request merged**